yt_dlp/extractor/franceculture.py

```diff
@@ -44,18 +54,42 @@ def _real_extract(self, url):
             ''',
             webpage, 'video data'))
 
-        video_url = video_data.get('data-url') or video_data['data-asset-source']
-        title = video_data.get('data-asset-title') or video_data.get('data-diffusion-title') or self._og_search_title(webpage)
-
+        title = video_data.get('data-emission-title') or video_data.get('data-diffusion-title') or self._og_search_title(webpage)
         description = self._html_search_regex(
             r'(?s)<div[^>]+class="intro"[^>]*>.*?<h2>(.+?)</h2>',
             webpage, 'description', default=None)
+
+        # page has playlist
+        if (re.search(r'<div[^>]+class="[^"]*?podcast-list[^"?]*?"[^>]*>', webpage) is not None):
```

Comment:
use search_html_regex (might have mixed the order but see in common.py)

Reply:
Not sure how `_html_search_regex` would be more useful as re.search is enough as the result is not reused

Reply:
The point of `_search_regex` and `_html_search_regex` is that it has proper error handling and standardizes the error messages